src/js/apps/patients/schedule/reduced-schedule/reduced_schedule_app.js

```diff
@@ -98,7 +98,8 @@ export default App.extend({
   },
   beforeStart() {
     const filter = this.getState().getEntityFilter();
-    return Radio.request('entities', 'fetch:actions:collection', { data: { filter } });
+    const fields = { flows: ['name', 'state'] };
```

Comment:
https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/schedule/schedule_app.js#L117

The pattern we're matching is here